content/en/tracing/llm_observability/sdk.md

```diff
@@ -346,6 +354,11 @@ def process_message():
     return 
 {{< /code-block >}}
 
+
+## Force flushing in serverless environments
+
+To gaurantee spans are sent in serverless environments, use the `LLMObs.force_flush()` function right before your application exits. This function blocks until all LLM Observability data has been submitted to the Datadog backend.
```

Comment:
```suggestion
To guarantee spans are sent in serverless environments, use the `LLMObs.force_flush()` function right before your application exits. This function blocks until all LLM Observability data has been submitted to the Datadog backend.
```

Reply:
Is a word between "blocks" and "until" missing? What does the function block? Or is it blocked?

Reply:
Just updated the description to clarify!